react-notes/src/components/Notepad/Notepad.jsx

```diff
@@ -1,22 +1,40 @@
 import PropTypes from 'prop-types';
 
-import { NO_SELECTED_NOTES } from '../../config/constants';
-import { WrapperStyled, NotepadStyled, NotepadTop, NotepadPapper } from './styled';
+import { WrapperStyled, NotepadStyled, NotepadTop, NotepadPapper, NotepadTitle } from './styled';
+import { NO_SELECTED_NOTES } from '../../config/constants/index';
+import ToolbarContainer from '../Toolbar/toolbarContainer';
 
-const Notepad = ({ note: { title, content, date } }) => {
+const Notepad = ({ note, isEdit, setEdit, setSave, setText, setTitle, onDelete }) => {
   return (
     <WrapperStyled className="wrapper">
-      <NotepadStyled className="notepad">
-        <NotepadTop className="notepad__title">
-          <h3>{title || null}</h3>
-          <span>{date || null}</span>
+      <NotepadStyled className="notepad" onClick={(e) => e.stopPropagation()}>
+        <NotepadTop className="notepad__top">
+          <NotepadTitle
+            className="notepad_title"
+            contentEditable={isEdit}
+            suppressContentEditableWarning
+            onInput={(e) => setTitle(e)}
+          >
+            {note ? note.title : null}

```

Comment:
Try to reduce an amount of `null` in code
```suggestion
            {note && note.title}

```